cmake/std/atdm/common/toss3/environment_new.sh

```diff
@@ -32,39 +31,17 @@ else
 fi
 
 # Common paths and modules for both intel-1{8,9}
-sparc_tpl_base=${ATDM_CONFIG_SPARC_TPL_BASE}
 module load cmake/3.12.2
 
-if [ "$ATDM_CONFIG_COMPILER" == "INTEL-18.0.2_OPENMPI-4.0.1" ]; then
-    module load intel/18.0.2.199
-    module load mkl/18.0.5.274 # Needed to address defect in MKL (#5316, #3992, #3914)
-    module load openmpi-intel/4.0
-
-    sparc_tpl_ext=cts1-bdw_intel-19.0.5
-    sparc_tpl_mpi_ext=cts1-bdw_intel-19.0.5_openmpi-4.0.1
-    # rabartl: ToDo: Above, we need to find a way to extract 'cts1-bdw' out of
-    # this file for this to be general!
-
-    export PATH=/usr/tce/packages/gcc/gcc-6.1.0/bin:${PATH}
-    export LD_LIBRARY_PATH=/usr/tce/packages/gcc/gcc-6.1.0/lib64:${LD_LIBRARY_PATH}
-elif [ "$ATDM_CONFIG_COMPILER" == "INTEL-19.0.5_OPENMPI-4.0.1" ]; then
-    module load intel/19.0.5.281
-    module load mkl/18.0.5.274 # Needed to find libmkl_intel_lp64.so (ATDV-212)
-    module load openmpi-intel/4.0
-
-    sparc_tpl_ext=cts1-bdw_intel-19.0.5
-    sparc_tpl_mpi_ext=cts1-bdw_intel-19.0.5_openmpi-4.0.1
-    # rabartl: ToDo: Above, we need to find a way to extract 'cts1-bdw' out of
-    # this file for this to be general!
-
-    export CBLAS_ROOT=/projects/global/toss3/compilers/intel/intel_2019/compilers_and_libraries_2019.5.281/linux
-    export COMPILER_ROOT=/projects/global/toss3/compilers/intel/intel_2017/compilers_and_libraries_2019.5.281/linux
-    export SPARC_HDF5=hdf5-1.10.5
+if [ "$ATDM_CONFIG_COMPILER" == "DEFAULT" ] ; then
+  export ATDM_CONFIG_COMPILER=INTEL-19.0.4_OPENMPI-4.0.3
+fi
```

Comment:
I think it is impossible to have `"$ATDM_CONFIG_COMPILER" == "DEFAULT"` due to the logic in the `ats1/custom_builds.sh` file.  Therefore, I think you can remove this if-statement.

Reply:
Addressed in https://github.com/e10harvey/Trilinos/pull/1